src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -18,13 +19,14 @@ export default class SaveInProgressIntro extends React.Component {
         const savedAt = this.props.lastSavedDate
           ? moment(this.props.lastSavedDate)
           : moment.unix(savedForm.last_updated);
+        const expirationDate = moment(savedAt).add(30, 'days');
```

Comment:
This is the data that's in the profile, so I think it's just partial info from the `metadata`. It might make sense to add `expires_at` to that data.